app/controllers/rents_controller.rb

```diff
@@ -0,0 +1,22 @@
+class RentsController < ApplicationController
+  respond_to :json
+  before_action :authenticate_user!
+
+  def index
+    rent = current_user.rents
+    if rent == []
+      render json: { error: 'The user has no rents' }, status: :not_found
```

Comment:
Try using early returns instead of `if ... else` blocks if it is possible

Reply:
Great! Done, thanks!